app/crud/comprehensive_evaluation.py

```diff
@@ -0,0 +1,100 @@
+from sqlalchemy.ext.asyncio import AsyncSession
+from app.models.evaluation import ComprehensiveEvaluation
+from uuid import UUID
+from typing import List
+from sqlalchemy import select, delete
+from app.schemas.comprehensive_evaluation import (
+    ComprehensiveEvaluationCreate,
+    ComprehensiveEvaluationResponse,
+    ComprehensiveEvaluationUpdate,
+)
+from app.utils.to_snake_case import camel_to_snake
+
+
+async def create_comprehensive_evaluation(
+    db: AsyncSession,
+    user_id: UUID,
+    comprehensive_evaluation: ComprehensiveEvaluationCreate,
+) -> ComprehensiveEvaluationResponse:
+
+    db_comprehensive_evaluation = ComprehensiveEvaluation(
+        user_id=user_id,
+        overall_achievement_rate=comprehensive_evaluation.overallAchievementRate,
+        evaluation_text=comprehensive_evaluation.evaluationText,
+        strength_achievement_rate=comprehensive_evaluation.strengthAchievementRate,
+        strength_text=comprehensive_evaluation.strengthText,
+        improvement_achievement_rate=comprehensive_evaluation.improvementAchievementRate,
+    )
+
+    db.add(db_comprehensive_evaluation)
+    await db.commit()
+    await db.refresh(db_comprehensive_evaluation)
+    return db_comprehensive_evaluation
+
+
+async def read_comprehensive_evaluations(
+    db: AsyncSession, user_id: UUID
+) -> List[ComprehensiveEvaluationResponse]:
+    comprehensive_evaluations = await db.execute(
+        select(ComprehensiveEvaluation).where(
+            ComprehensiveEvaluation.user_id == user_id
+        )
+    )
+    return comprehensive_evaluations.scalars().all()
+
+
+async def read_comprehensive_evaluation(
+    db: AsyncSession, user_id: UUID, comprehensive_evaluation_id: UUID
+) -> ComprehensiveEvaluationResponse:
+    comprehensive_evaluation = await db.execute(
+        select(ComprehensiveEvaluation).where(
+            ComprehensiveEvaluation.id == comprehensive_evaluation_id,
+            ComprehensiveEvaluation.user_id == user_id,
+        )
+    )
+
+    result = comprehensive_evaluation.scalars().first()
+
+    return result
```

Comment:
다른 코드와 일관성 유지를 위해 아래처럼 수정하면 좋을 것 같습니당

```suggestion
    return comprehensive_evaluation.scalars().first()
```

Reply:
@GideokKim 
이 부분 수정 완료 했습니다!